src/Tribe/Service_Providers/Context.php

```diff
@@ -44,32 +45,34 @@ public function register() {
 	 * @return array The modified context locations.
 	 */
 	public function filter_context_locations( array $locations = [] ) {
-		$locations = array_merge(
+		$view_query_var = View::query_var();
+
+		$locations      = array_merge(
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 6 spaces

<sub>hash: 55eab2919f9fb38f7e28b443283d84a0</sub>